### ISSUE TITLE
Details on distillation

### ISSUE THREAD

**Issue by jinhong-ni** [modelscope/DiffSynth-Studio#893]
Hi there, thanks so much for releasing and open-sourcing your great models!

I'm wondering if you could disclose more details on how `Qwen-Image-Distill-Full` is distilled, in particular, what distillation algorithms was used (like LCM, CTM, DMD, etc)? We are using this in a research project and want to ensure that we refer to your work accurately. Thanks so much!

**Comment by 1343744768** [comment]
+1 #861

**Comment by Artiprocher** [comment]
@jinhong-ni Our training code has been released:

https://github.com/modelscope/DiffSynth-Studio/blob/main/examples/qwen_image/model_training/lora/Qwen-Image-Distill-LoRA.sh

But you may be disappointed — we did not use any distillation algorithms at all. Below is the pseudocode for distillation training:

```python
with torch.no_grad():
    image1 = pipe(prompt, steps=40, cfg=4, return_latents=True)
image2 = pipe(prompt, steps=4, cfg=1, return_latents=True)
loss = mse(image1, image2)
```

**Comment by jinhong-ni** [comment]
Thanks @Artiprocher for this insight!

**Comment by dongdk** [comment]
> [@jinhong-ni](https://github.com/jinhong-ni) Our training code has been released:
> 
> https://github.com/modelscope/DiffSynth-Studio/blob/main/examples/qwen_image/model_training/lora/Qwen-Image-Distill-LoRA.sh
> 
> But you may be disappointed — we did not use any distillation algorithms at all. Below is the pseudocode for distillation training:
> 
> with torch.no_grad():
>     image1 = pipe(prompt, steps=40, cfg=4, return_latents=True)
> image2 = pipe(prompt, steps=4, cfg=1, return_latents=True)
> loss = mse(image1, image2)

加上一些感知loss，重构loss会不会有效果提升？
